README.md

```diff
@@ -3,7 +3,7 @@
 [![Build Status](https://devdiv.visualstudio.com/DevDiv/_apis/build/status/Oryx/Oryx-BuildImage?branchName=master)](https://devdiv.visualstudio.com/DevDiv/_build/latest?definitionId=9922?branchName=master)
 
 Oryx is a build system which automatically compiles source code repos into
-runnable artifacts. It is used to build web apps for [Azure App Service][] and
+runnable artifacts. It is used to build web apps for [Azure App Service](https://docs.microsoft.com/en-us/azure/app-service/configure-common) and
```

Comment:
this is not needed, since "[Azure App Service]: https://azure.microsoft.com/services/app-service/" is defined below.